runtime/protocol/http-test/build.gradle.kts

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0.
+ */
+description = "Utilities for testing HTTP requests"
```

Comment:
note

at some point maybe we want to add some kind of post build test that validates that none of our test modules accidentally made it into our rt or service artifacts, unless there is already a mechanism in place that guards this?

Reply:
If it's not a runtime dependency of them then we are good. I'm not sure of a mechanism for it but the only time you would need it is testing so you would have to accidentally add it to say `commonMain` instead of `commonTest` source set.